### PR TITLE
Om refactor journey component

### LINKED ISSUE #119: Add information to Journey component

### PR DESCRIPTION
**Pull request by Atos20**
### What does this PR do?
- Updates Journey component to display app instructions
- Changes the style of the Journey component
- Displays 3 cards to show important information about how to use the application

### Where should the reviewer start?
- `src/component/Journey/Journey.css`
- `src/component/Journey/Journey.tsx`

### How should this be manually tested?
run `npm start` then navigate to the Journey component

### Any background context you want to provide?

Previously the `Journey` component introduced the avatar 'Teki' to the user but there was not enough information on the page for the user to fully understand the application. These PR changes try to solve the issue by making the page more dynamic and give the user more information about what to expect when using the application. 

### What are the relevant tickets?
closes #119 
#115
### Screenshots (if appropriate)

Screen Shot 2021-01-26 at 3 43 47 PM

### CODE REVIEW

**Review by BlakeDonn — APPROVED** [review]
Love the design, the tests should be decently easy to fix if we get to that point

File: src/components/Journey/Journey.css [hunk 1]
```diff
@@ -1,60 +1,29 @@
...
-  grid-template-rows: repeat(5, 1fr);
 }
 
-.information-container{
-  grid-area: 1/ 1/ 3 / 3;
-  display: inline-flex;
+.container{
+  height: 79vh;
+  display: flex;
   justify-content: center;
```
Comment (Arique1104) [inline src/components/Journey/Journey.css:9]: What is this file in charge of?

**Review by Arique1104 — APPROVED** [review]
During standup, I'd love to review these changes, if just to see what they looking like visually and what the significance of each change means.

File: src/components/Journey/Journey.css (hunk 1 above)
Comment (timkeresey) [reply src/components/Journey/Journey.css:9]: This is the CSS file for the styling of that component

**Review by timkeresey — APPROVED** [review]
The style looks great!

**Pull request merged**